### ISSUE TITLE
Gcode Documentation post processing script not working with Cura 5.x

### ISSUE THREAD

**Issue by csanii123** [5axes/Cura-Postprocessing-Scripts#9]
Not sure if I am doing something wrong. I put the script in the scripts directory but it doesn't give the same output as it did in Cura v4.13.1
I attached a screenshot of the v5.4 window showing the plugin active as well as the v4.13.1 and v5.4 outputs.
The 4.13.1 output gives all the settings in plain english while the v5.4 doesn't.
I tried v5.3 as well and checked my gcode files and ever since v5.0 it looks like it went missing. 
I am using Win11 Pro 64BIt v22H2
Can you assist ?

![Cura v5 4 Screenshot#2](https://github.com/5axes/Cura-Postprocessing-Scripts/assets/67832530/8cc0e077-922c-4628-9f32-86c9020af51e)
![Cura v4 13 1 Screenshot](https://github.com/5axes/Cura-Postprocessing-Scripts/assets/67832530/86e66f10-080d-4d14-81de-f8778198230b)
![Cura v5 4 Screenshot#1](https://github.com/5axes/Cura-Postprocessing-Scripts/assets/67832530/f5361873-2e0b-4cfc-811d-34cf18ea5e6f)

**Comment by 5axes** [comment]
Modifications in the API code of Cura.. Must be modified

**Comment by csanii123** [comment]
WOW! I just downloaded the v5.4 updated version and it looks to be working. Thanks for fixing it and for doing it so fast !
